### PR TITLE
shadowmario reminded me to keep myself safe such a kind person

### PR DESCRIPTION
**Pull request by ghost**
.

### CODE REVIEW

File: source/options/NotesSubState.hx [hunk 1]
```diff
@@ -277,4 +293,4 @@ class NotesSubState extends MusicBeatSubstate
 		item.offset.x = (40 * (item.lettersArray.length - 1)) / 2;
 		if(roundedValue < 0) item.offset.x += 10;
 	}
-}
\ No newline at end of file
+}
```
Comment (hayfii) [inline source/options/NotesSubState.hx:296]: i probably made some mistakes on the code

**Comment by ShadowMario** [comment]
kys

**Comment by hayfii** [comment]
> kys

Thanks for reminding me to keep myself safe ShadowMario ❤

**Comment by CrusherMods** [comment]
such a kind person

**Comment by hayfii** [comment]
> kys

i know this is dumb but it looks dead without tweens 🤓

**Pull request closed without merging**